src/plotman/log_parser.py

```diff
@@ -0,0 +1,140 @@
+import re
+from plotman.plotinfo import PlotInfo
+
+class PlotLogParser:
```

Comment:
Alternatively, maybe it would make sense to hold onto these objects with them holding the open files and then we can incrementally parse the logs instead of reparsing them every loop, once this is used for the main job info log parsing.  At least I think that's what is done now, I didn't check back right now...